### PR TITLE
Milestone Two: Basic Components

### PR DESCRIPTION
**Pull request by Zubenna**
# In this PR, I implemented the following;
- Created Button, Display and ButtonPanel Components.
- I implemented the components.
- I styled the components and display the first version of the calculator UI.
- Pushed update to Heroku.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @Zubenna  👋

**You've done pretty well here with obvious attention to the project requirements. 👌**

However, let's make a couple of changes.

- [ ] It's important that the project is properly linted. To that effect please follow the linting instructions [here](https://github.com/microverseinc/linters-config/tree/master/react-redux) and ensure that both `.eslintrc.json` and `.stylelintrc.json` files are included in the project.

- Please refer to other comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+function App() {
+  return (
+    <div className="App">
+      <h1>Adding Basic Components: Displaying Calculator UI Version 1</h1>
+      <section className="calc-container">
+        <Display result="0" />
+        <ButtonPanel />
+      </section>
+    </div>
```
Comment (mikenath223) [inline src/components/App.js]: - [ ] Kindly wrap both Display and ButtonPanel with the [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax. Its not necessary to add an extra DOM node this is what React.Fragment syntax achieves.

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+const isEqual = val => val === '=';
+
+const Button = ({ name }) => (
+  <button type="button" className={isEqual(name) ? 'eqlBtn' : 'btn'}>
+    { name }
+  </button>
+);
+
+Button.propTypes = {
+  name: PropTypes.node.isRequired,
```
Comment (mikenath223) [inline src/components/Button.js]: - [ ] 'name' should be a prop of the type string, not a node. Kindly update the proptype to string.

File: src/components/ButtonPanel.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      <Button name="2" />
+      <Button name="3" />
+      <Button name="+" />
+    </div>
+    <div className="row">
+      <Button name="0" />
+      <Button name="." />
+      <Button name="=" />
+    </div>
+  </>
```
Comment (mikenath223) [inline src/components/ButtonPanel.js:35]: - [ ] Nice idea using the React.Fragment short syntax here.

File: src/components/Display.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ result }) => (
+  <input className="display" type="text" value={result} />
+);
+
+Display.propTypes = {
+  result: PropTypes.node.isRequired,
```
Comment (mikenath223) [inline src/components/Display.js]: - [ ] Also the 'result' should be a prop of the type string, not a node. Kindly update the proptype to string.

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****


 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**